### ISSUE TITLE
'find' is not defined error when initializing GooglePay (MS Edge)

### ISSUE THREAD

**Issue by simeyla** [braintree/braintree-web#446]
### General information

* SDK version: 3.48.0
* Environment: Production
* Browser and OS : Windows Edge 42

### Issue description

Just did a soft launch of our site (finally!) and seeing the following stack trace for Google Pay in Edge. This is in the initialization phase - not a payment attempt.

The stack trace below was logged from a real customer, but I was able to repeat it by just opening my checkout (completely custom) in Edge.

```
ReferenceError: 'find' is not defined
at e.exports (https://www2.example.com/14.365a3634db45ac4f6215.js:1:832033) 
at f.prototype._getDefaultConfig (https://www2.example.com/14.365a3634db45ac4f6215.js:1:785908)
at f.prototype._createV2PaymentDataRequest (https://www2.example.com/14.365a3634db45ac4f6215.js:1:786358)
at f.prototype.createPaymentDataRequest (https://www2.example.com/14.365a3634db45ac4f6215.js:1:786734)
...
```

This customer *did* go on to actually successfully place an order using Credit Card - not GooglePay. So definitely the rest of the Braintree API is working - just this 'find' function is causing problems. 

So I'm quite puzzled how only the `find` function is unavailable. My only guess is that it's something to do with it having a naming clash - and Edge thinks it's referring to a built-in function? Maybe it just needs renaming?
 

`Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36 Edge/17.17134
`

This code is the actual code that fails with `'find' is not defined`

**My code**

    const allowedPaymentMethods = googlePaymentInstance.createPaymentDataRequest().allowedPaymentMethods;`

**Your Code**

```
 if (googleMerchantId) {
      data.merchantInfo = {
        merchantId: googleMerchantId
      };
    }

    if (configuration.authorizationType === 'TOKENIZATION_KEY') {
      allowedPaymentMethod = find(data.allowedPaymentMethods, 'type', 'CARD');

      if (allowedPaymentMethod) {
        allowedPaymentMethod.tokenizationSpecification.parameters['braintree:clientKey'] = configuration.authorization;
      }
    }
```

**Comment by cooperbarth** [comment]
Thanks for bringing this to our attention! We have fixed the issue, and it will be available in the next release. (Coming soon to an SDK near you!)

**Comment by crookedneighbor** [comment]
This is fixed in v3.50.1. Thanks for the report!
